src/zzz_od/api/bridges.py

```
@@ -79,3 +92,163 @@ def _detach() -> None:
+            """向相关通道广播事件"""
+            try:
+                import asyncio
+                message = {
+                    "type": event_type,
+                    "data": data,
+                    "timestamp": datetime.now().isoformat(),
+                    "task_id": self.run_id
+                }
+
+                # 向任务专用通道广播
+                asyncio.create_task(manager.broadcast_json(task_channel, message))
```

Ruff (0.12.2)

Store a reference to the return value of `asyncio.create_task`

(RUF006)